etc/s6/services/nginx/run

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/execlineb -P
+if { test -S /var/run/php-fpm.sock }
```

Comment:
Is this really the full syntax?
In https://github.com/PrivateBin/docker-nginx-fpm-alpine/issues/42 you said you added some shell script instead?

Reply:
Yeah, I'll extend the scriptlet `test -S /var/run/php-fpm.sock || (echo "waiting on /var/run/php-fpm.sock" >&2 && sleep 1 && exit 1)` from there a little bit. It should be possible to wait in a while loop (ideally with no delay if the socket already exists), but time out after i.e. 15 seconds if the php process never came up, so the container run gets failed and the admin can handle it.

Reply:
I added the sleep script, to implement a sleep; until I found out that s6 will retry every second when then run script returns an error. 

When the socket does not appear, I trust on php-fpm producing appropriate errors.